ios/Classes/FlutterWebviewPlugin.m

```diff
@@ -235,6 +266,19 @@ - (void)cleanCookies {
 }
 
 #pragma mark -- WkWebView Delegate
+
+- (WKWebView *)webView:(WKWebView *)webView createWebViewWithConfiguration:
+(WKWebViewConfiguration
+*)configuration forNavigationAction:(WKNavigationAction *)navigationAction windowFeatures:(WKWindowFeatures *)windowFeatures {
+     UIApplication *application = [UIApplication sharedApplication];
+    if (@available(iOS 10.0, *)) {
+        [application openURL:navigationAction.request.URL options:@{} completionHandler:nil];
+    } else {
+        // You're screwed
+    }
```

Comment:
Please remove empty else / add some message. 

Also, from what I see this api is available from 9.0+ https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623112-application?language=occ